src/sql/parts/notebook/notebook.component.ts

```diff
@@ -331,6 +337,47 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	// We can keep this method in common utilities
+	private getFilePathFromRecentWorkspace(untitledResource: URI): string {
```

Comment:
Should you move this to utilities now then?

Reply:
It's got the dependencies from History Service which may not be a good way to put it in utils.